src/main/java/uk/gov/hmcts/reform/unspec/model/OrganisationPolicy.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.unspec.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OrganisationPolicy {
+
+    @JsonProperty("Organisation")
+    private Organisation organisation;
```

Comment:
Could we name this type OrganisationId or something to stop the clash with Organisation type when used?